pkg/getter/getter.go

```diff
@@ -183,16 +185,26 @@ const (
 	// The cost timeout references curl's default connection timeout.
 	// https://github.com/curl/curl/blob/master/lib/connect.h#L40C21-L40C21
 	// The helm commands are usually executed manually. Considering the acceptable waiting time, we reduced the entire request time to 120s.
-	DefaultHTTPTimeout = 120
+	defaultHTTPTimeout = 120
```

Comment:
```suggestion
	DefaultHTTPTimeout = 120
```

Since this is not an internal package, this probably needs to stay public or this would be a breaking change.